source/en/ndr.rst

```diff
@@ -272,12 +272,14 @@ The model has options to calculate nitrogen, phosphorus, or both. You must provi
     .. note::
        Loads are the sources of nutrients associated with each LULC class. This value is the total load from all sources. If you want to represent different levels of fertilizer application, you will need to create separate LULC classes, for example one class called "crops - high fertilizer use" a separate class called "crops - low fertilizer use" etc.
 
+    - :investspec:`ndr.ndr biophysical_table_path.columns.nut_load_type`
+
     .. note::
-       Data sources may provide loading values as either the amount of applied nutrient (e.g. fertilizer, livestock waste, atmospheric deposition); or as “extensive” measures of contaminants, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.) In the case of having applied nutrient values, they should be corrected for the nutrient retention provided by the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type:
+       Data sources may provide loading values as either the amount of applied nutrient (e.g. fertilizer, livestock waste, atmospheric deposition); or as “extensive” measures of contaminants, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.). The two types are denoted *application-rate* and *measured-runoff* respectively in the biophysical table. In the case of having applied nutrient values, the model will correct for the nutrient retention provided by the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type:
 
        applied_nutrient * (1 - retention_efficiency)

```

Comment:
I'd like to make some clarifying edits to this paragraph and equation so the two loading options, and the words and fields that are used to describe them, are clear and consistent. One example of this would be changing the equation to something like

> _loading = application rate * (1 - retention efficiency)_
> Using the specific biophysical table fields:
> _loading = application-rate * (1 - [eff_n or eff_p])_